db/migrate/20170531094819_move_conferences_to_organizations.rb

```diff
@@ -0,0 +1,21 @@
+class MoveConferencesToOrganizations < ActiveRecord::Migration
+  class TempConference < ActiveRecord::Base
+    self.table_name = 'conferences'
+  end
+
+  class TempOrganization < ActiveRecord::Base
+    self.table_name = 'organizations'
+  end
+
+  def change
+    add_reference :conferences, :organization, index: true
+    add_foreign_key :conferences, :organizations, dependent: :delete
```

Comment:
Yes, the `foreign_key` option should work. I was actually thinking about the `null: false` option. We should have it, but I don't think it will work if the DB already has conferences.
I'm thinking that if the database already contains conferences, upon creating the `organization_id` field, it will be NULL, but we should not allow NULL values (hence we should use `null: false`). But then the field won't be created. So we need to create it first without the `null: false` option, fill it with the default organization's id, and then add the `null:  false` option.